cmd/ci-operator-checkconfig/main.go

```diff
@@ -49,7 +49,8 @@ func (o *options) parse() error {
 	if err := o.loadResolver(registryDir); err != nil {
 		return fmt.Errorf("failed to load registry: %w", err)
 	}
-	ciOPConfigAgent, err := agents.NewConfigAgent(o.ConfigDir, agents.WithOrg(o.Org), agents.WithRepo(o.Repo))
+	// TODO: add error handling
```

Comment:
This program only loads the configuration once, we don't care if it's changed later.

Since the initial loading already validates that configuration files are valid, we don't need to do anything else here.  This `TODO` can be removed.